block_designs.tex

```diff
@@ -263,20 +401,26 @@ \section{Cyklické blokové plány a diferenčné množiny}
 
 \end{theorem_hard}
 
-\begin{remark}
-
-Existujú aj ďalšie triedy diferenčných množín, napríklad bikvadratické rezíduá alebo tzv. \emph{twin prime power difference set}.
-
-\end{remark}
-\begin{toreview}
+\begin{theorem_hard}{(Typ B dif. množín --- bikvadratické rezíduá)}\\
+	Nech $F := GF(q)$ je konečné pole, kde $q = p^l$ a $p = 4x^2 + 1$ pre nepárne $x$. 
+	Nech $r$ je generátor grupy $F = (GF(q), +)$. 
+	Potom množina bikvadratických rezíduí grupy $F$ $BQR(F) := \set{TODO}$
+	je $(v, k, \lambda)$-diferenčnou množinou s parametrami:
+	
+	$$v = p = 4x^2+1, k = x^2, \lambda = \frac{x^2 - 1}{4}$$
+	
+	\end{theorem_hard}
 
-\todo[inline]{rozpisat bikvadraticke rezidua (neviem to najst v literature)}
-\todo[inline]{rozpisat twin prime power (neviem to najst v literature)}
-\end{toreview}
+\begin{theorem_hard}{(Typ T dif. množín --- Twin prime power dif. množiny)}\\ 
+	Nech je množina $D$ v grupe $((GF(q), +) \times (GF(q + 2), +), +)$, kde $q$ aj $q + 2$ sú mocniny prvočísel, definovaná nasledovne $D = \{(x, y) ~|~ y = 0$ alebo x aj y sú nunulové a oba sú súčasne štvorce alebo súčasne neštvorce$\}$. Potom $D$ je $(v, k, \lambda)$-diferenčná množina s parametrami:
+	$$v = q^2 + 2q, k = \frac{q^2 + 2q - 1}{2}, \lambda = \frac{q^2 + 2q - 3}{4}$$
```

Comment:
It'd be nice to add spaces or maybe even `\quad` after commas.

Reply:
fixed in every diff. set type